love/src/components/EnvironmentSummary/EnvironmentSummary.jsx

```diff
@@ -132,6 +133,7 @@ export default class EnvironmentSummary extends Component {
       windSpeed,
     } = this.props;
     const { hideIconTemperature } = this.state;
+    const degradation = 'Unknown';
```

Comment:
I think if you are setting degradation = 'Unknown' on the selectors, it should arrive here with the same value.